### ISSUE TITLE
context for (if or when) we add future commands to Kyogas.

### ISSUE THREAD

**Issue by tuxza** [pixelqualitysoftware/kyogas#12]
As centurion said:
I designed it to accept a command as ARGV[0], which we have somehow not implemented yet. When Kyogas becomes more complex, it won't have only one use; therefore just invoking kyogas and a filename will cause ambiguity. The better solution is to route the users command (argv[0])

_Originally posted by @Centurion774477 in https://github.com/pixelqualitysoftware/kyogas/pull/11#issuecomment-5751354830_

as of writing this. The current version is jusy
kyogas file.kyo

a future version should be 
kyogas parse file.kyo

since we only have one feature so far. It would make the most sense to run the code and pass the file directly.

**Comment by werSquared** [comment]
> "As of writing this the current version is just kyogas file.kyo"

no, it's not, that's the whole problem.
if you try that the program thinks you're missing an argument

**Comment by Centurion774477** [comment]
Sorry but how haven't we discovered this problem earlier? It's been over two weeks since I wrote that code and it's pretty important.

**Comment by tuxza** [comment]
> > "As of writing this the current version is just kyogas file.kyo"
> 
> no, it's not, that's the whole problem.
> if you try that the program thinks you're missing an argument 

I meant dotnet run insertthinghere file.kyo

plex made it dotnet run file.kyo I believe

**Comment by tuxza** [comment]
> Sorry but how haven't we discovered this problem earlier? It's been over two weeks since I wrote that code and it's pretty important.

It was really confusing when I tried using kyogas the first time 😭
